src/components/trends/smbg/FocusedSMBGPointLabel.js

```diff
@@ -57,7 +65,11 @@ const FocusedSMBGPointLabel = (props) => {
   const pointTooltips = _.map(allSmbgsOnDate, (smbg, i) => (
     <Tooltip
       key={i}
-      content={<span className={styles.number}>{displayBgValue(smbg.value, bgUnits)}</span>}
+      content={
+        <span className={styles.number}>
+          {displayBgValue(smbg.value, bgUnits, getOutOfRangeThreshold(smbg))}
```

Comment:
This is the place where I'm least sure of my approach... Because here we *are* dealing with a single data point, so we could shortcut the somewhat annoying need to extract and reshape the relevant threshold (if any) with `getOutOfRangeThreshold` by just doing something like:
```
const outOfRangeAnnotation = _.find(
  smbg.annotations || [], (annotation) => (annotation.code === 'bg/out-of-range')
);
...then inside the <span>...
{outOfRangeAnnotation ? outOfRangeAnnotation.value : displayBgValue(smbg.value, bgUnits)}
```

The problem I think is that the aggregate data and the individual datums really would each be best suited by a `displayBgValue` function with a different interface. Ideally the display function for the individual datum BG values could take the whole datum and extract the annotations itself... In fact, in older tideline code [the equivalent of the `displayBgValue` function](https://github.com/tidepool-org/tideline/blob/master/js/data/util/format.js#L26-L40) took the entire datum in its params for precisely this reason. So it's a tough problem I think...